lib/Onnxifi/CMakeLists.txt

```diff
@@ -3,7 +3,11 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR})
 add_library(onnxifi-glow
             SHARED
               Base.cpp
-              onnxifiGlow.cpp)
+              onnxifiGlow.cpp
+              ThreadPool.cpp)
+
+add_library(onnxifi-glow-thread-pool
```

Comment:
if you expose thread pool as a library then it's a bit weird that `ThreadPool.cpp` is also included in `onnxifi-glow` library.

I think it's OK to keep ThreadPool.cpp just in `onnxifi-glow`. If there will be other use cases for ThreadPool we could extract that into the library.